src/test/java/uk/gov/hmcts/cmc/claimstore/services/ResponseDeadlineCalculatorTest.java

```diff
@@ -86,11 +86,14 @@ public void calculateResponseDeadlineWhenResponseDateOnGoodFridayThenReturnFirst
     @Test
     public void calculatePostponedResponseDeadlineWhenNoBankHolidays() {
 
-        LocalDate responseDeadline = calculator.calculateResponseDeadline(toDate("2017-05-08"));
-        LocalDate postponedDeadline = calculator.calculatePostponedResponseDeadline(responseDeadline);
+        LocalDate issuedOn = toDate("2017-05-08");
+        LocalDate responseDeadline = calculator.calculateResponseDeadline(issuedOn);
+        LocalDate postponedDeadline = calculator.calculatePostponedResponseDeadline(issuedOn);
+        LocalDate expectedIssueDate = toDate("2017-05-30"); // Tue as Mon was Bank holidays
```

Comment:
think the test name might need an update

Reply:
Yes as 2015-05-29 is a bank holiday, plus this test is being repeated below